boot/zephyr/Kconfig

```diff
@@ -68,6 +68,7 @@ config SINGLE_APPLICATION_SLOT
 
 choice
 	prompt "Signature type"
+	default BOOT_SIGNATURE_TYPE_ECDSA_P256 if HAS_HW_NRF_CC310
```

Comment:
We could select using `CC310` by default for `BOOT_SIGNATURE_TYPE_ECDSA_P256` if available. Currently tinycrypt is used by default (even if the board supports `CC310`).

Reply:
I'm not sure I understand what you mean. Do you mean that CC310 should be the backend for all types of signature types when the HW is available?

Reply:
> I'm not sure I understand what you mean. Do you mean that CC310 should be the backend for all types of signature types when the HW is available?

He mentioned `BOOT_SIGNATURE_TYPE_ECDSA_P256`, not all types.

Reply:
Sorry if something was not clear:
If CC310 HW is available, we can use CC310 as a default backend for `BOOT_SIGNATURE_TYPE_ECDSA_P256 `:

```
if BOOT_SIGNATURE_TYPE_ECDSA_P256
choice
	prompt "Ecdsa implementation"
	default BOOT_CC310 if HAS_HW_NRF_CC310
	default BOOT_TINYCRYPT
```
This is already added to nrf fork of MCUBoot bootloader repository.

Reply:
As far as I can see this is what we will get now in both upstream and downstream. This due to what is shown here:

```
config BOOT_CC310
	bool "Use CC310"		bool "Use CC310"
	select BOOT_USE_NRF_CC310_BL if HAS_HW_NRF_CC310		select BOOT_USE_NRF_CC310_BL if HAS_HW_NRF_CC310
	select NRF_CC310_BL if HAS_HW_NRF_CC310		select NRF_CC310_BL if HAS_HW_NRF_CC310
	select NRFXLIB_CRYPTO if SOC_FAMILY_NRF		select NRFXLIB_CRYPTO if SOC_FAMILY_NRF
	select BOOT_USE_CC310		select BOOT_USE_CC310
```

Reply:
I see your point, check it now @MarekPieta !

Reply:
Thank you. Approved.